src/resources/postcss/views/full/_single-event.pcss

```diff
@@ -234,7 +234,6 @@
 
 	/* TODO: move .event-tickets declaration to ET in the future if still needed,
 	/* depending on template updates */
-	&:before,
```

Comment:
Removing this will remove it for all themes, not just Avada. If this is an Avada specific issue, create a theme override at the bottom of this file for Avada overrides.